src/qt/networkstyle.cpp

```diff
@@ -71,8 +71,8 @@ NetworkStyle::NetworkStyle(const QString &_appName, const int iconColorHueShift,
         pixmap.convertFromImage(img);
     }
 
-    appIcon             = QIcon(pixmap);
-    trayAndWindowIcon   = QIcon(pixmap.scaled(QSize(256,256)));
+    appIcon = QIcon(pixmap);
+    trayAndWindowIcon = QIcon(pixmap.scaled(QSize(64, 64), Qt::KeepAspectRatio, Qt::SmoothTransformation));
```

Comment:
Would keeping the size of 256x256 (for all current and future HiDPI-ish situations) change the results?

Reply:
yea I'd say NACK on going back to 64x64 here, this makes assumptions about the size that the OS will use, and 256x256 has been more or less the default for OSX for a while. It's up to the OS to scale this image down.

Scaling to 256x256 likely has no effect because the input image is 256x256?